app/models/post.rb

```diff
@@ -0,0 +1,11 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] - [ ] Let's change the association between a user and a post to suit the project requirement. The ERD shows `author_id` instead of `user_id`. We want a post to belong to an `author`.
HINT: `belongs_to :author, class_name: 'User'`

Reply:
Thanks, done!